tests/test_config.py

```diff
@@ -61,84 +127,117 @@ def mock_default_config_filepath() -> Path | None:
     )
 
 
-def test_invalid_path() -> None:
-    with pytest.raises(ValueError):
-        _ = AnExample1Config.get(
-            reload=True, configfile_path='fi:\0\\l*e/p"a?t>h|.t<xt'
-        ).section1.field1
+def test_get(monkeypatch: pytest.MonkeyPatch, toml_file: Path) -> None:
+    AnExample1Config.set_filepath(toml_file)
+    assert AnExample1Config.get(reload=True).section1.field1 == "f1"
+    assert AnExample1Config.get().section1.field2 == 22
 
+    # test that by default it is not reloaded
+    if sys.version_info >= (3, 10):
 
-def test_get(monkeypatch: pytest.MonkeyPatch) -> None:
-    def mock_default_config_filepath() -> Path | None:
-        return Path(__file__)
+        def mock_tomllib_load(
```

Comment:
For these very simple methods, should we not just have no typing information? The split implementations to me only add confusion.

Reply:
Made the type equal to Any to accomplish the same